packages/framework/src/Controller/Admin/SliderController.php

```diff
@@ -78,7 +78,8 @@ public function listAction()
             ->from(SliderItem::class, 's')
             ->where('s.domainId = :selectedDomainId')
             ->setParameter('selectedDomainId', $this->adminDomainTabsFacade->getSelectedDomainId())
-            ->orderBy('s.position');
+            ->orderBy('s.position')
+            ->addOrderBy('s.id');
```

Comment:
is it necessary to order by id as a second ordering? 
this will take place only when several new slider items are created without reordering (position is then null). 

Wouldn't be better to recalculate position after creating a new item? 
(right now items with null in position behave differently in administration and on frontend)